### ISSUE TITLE
[AIR] linux://doc:source/ray-air/doc_code/computer_vision is flakey

### ISSUE THREAD

**Issue by ArturNiederfahrenhorst** [ray-project/ray#35551]
### What happened + What you expected to happen

<img width="1489" alt="Screenshot 2023-05-19 at 09 03 14" src="https://github.com/ray-project/ray/assets/9356806/4ca2833f-7b1b-461a-8ea0-522cc3db7ea2">

### Versions / Dependencies

master / 2.5.0 release branch

### Reproduction script

. 

### Issue Severity

None

**Comment by can-anyscale** [comment]
Since this is a release-blocker issue, please **close it only after the cherry pick fix is merged into 2.5 release branch**. 

Please add @ArturNiederfahrenhorst as one of the reviewer of the fix as well for tracking purpose. Thankks!

**Comment by gjoliver** [comment]
this may just be flaky. retrying.

**Comment by ArturNiederfahrenhorst** [comment]
@gjoliver You have to retry pretty hard to get this test green 😄 
Can I ignore this branch for the release and make this non-blocking?

CC @zhe-thoughts

**Comment by can-anyscale** [comment]
Probably up to the test owner to decide if this flakiness should be release-blocking, what do you think @gjoliver

**Comment by can-anyscale** [comment]
Fix now \o/
